examples/graceful_shutdown.py

```diff
@@ -5,46 +5,51 @@
 from contextlib import suppress
 from datetime import datetime
 from functools import partial
+from typing import Any, Callable, Dict, Optional
 
 from aiohttp import web
 
 from aiohttp_sse import EventSourceResponse, sse_response
 
-streams = web.AppKey("streams", weakref.WeakSet[EventSourceResponse])
-worker = web.AppKey("worker", asyncio.Task[None])
+streams_key = web.AppKey("streams_key", weakref.WeakSet["SSEResponse"])
+worker_key = web.AppKey("worker_key", asyncio.Task[None])
 
 
 class SSEResponse(EventSourceResponse):
     async def send_json(
         self,
-        data,
-        id=None,
-        event=None,
-        retry=None,
-        json_dumps=partial(json.dumps, indent=2),
-    ):
+        data: Dict[str, Any],
+        id: Optional[str] = None,
+        event: Optional[str] = None,
+        retry: Optional[int] = None,
+        json_dumps: Callable[..., str] = partial(json.dumps, indent=2),
```

Comment:
I think this must be 1 parameter.
```suggestion
        json_dumps: Callable[[Any], str] = partial(json.dumps, indent=2),
```

Reply:
Provided partial already has 2 params and mypy shouldn't pass this

Reply:
I think you've misunderstood this. We call `json_dumps` in one place, with 1 argument. Therefore, the provided function _must_ accept 1 positional parameter and work. That's exactly what my suggested change requires. The partial is compatible with this. Disallowed function are those which have 0 positional parameters, or those which have more than 1 _required_ positional parameter.